woocommerce/woocommerce-android
# Connect to reader from collect payment flow

Parent issues #3725 and #3726

This PR adds support for redirecting to connect-to-reader flow when a user tries to collect a payment and no reader is connected.

To test
- HW reader is not supported for this flow -> simulated reader will be used by default
- Testing site with WCPay installed and activated is required
1. Open detail of an order in US dollars
2. Tap on "Collect payment"
3. Notice "connect-to-reader" flow is started
4. Tap on "Connect to reader"
5. Notice when the connection flow completes, you are automatically redirected to collect payment flow
6. Finish the payment and dismiss the payment dialog
7. Tap on "Collect payment"
8. Notice you are redirected to payment flow since a reader is already connected

Update release notes:
